installation/sql/postgresql/joomla.sql

```diff
@@ -501,6 +502,7 @@ INSERT INTO "#__extensions" ("extension_id", "name", "type", "element", "folder"
 (30, 'com_contenthistory', 'component', 'com_contenthistory', '', 1, 1, 1, 0, '', '', '', '', 0, '1970-01-01 00:00:00', 0, 0),
 (31, 'com_ajax', 'component', 'com_ajax', '', 1, 1, 1, 1, '', '', '', '', 0, '1970-01-01 00:00:00', 0, 0),
 (32, 'com_postinstall', 'component', 'com_postinstall', '', 1, 1, 1, 1, '', '', '', '', 0, '1970-01-01 00:00:00', 0, 0);
```

Comment:
ok install on postgresql will be broken ... see the last character in this line.